docs/proposal/terms_and_abbreviations.tex

```diff
@@ -4,18 +4,28 @@ \section*{Terms and Abbreviations}
 
 Technical terms and abbreviations dictionary go here.
 
-\begin{tabular}[h]{rl}
+\begin{tabular}[h]{rp{0.75\linewidth}}
     \hline
     \textbf{Term} & \textbf{Definition}\\
     \hline
 
-    NN &\\
-    CNN &\\
-    RNN &\\
-    YOLO &\\
-    TX & Transmitter.\\
+    ANN & Artificial Neural Network, or simply ``Neural Network'', is a data processing model modeled after neuron interactions. The process consists of forward propagation using several matrix multiplications.\cite{ann}\\
+    ASIC & Application-specific Integrated Circuits.\\
+    CNN & Convolutional neural network are neural networks that is especially useful for image classification.\cite{cnn} \\
+    ECE & Department of Electrical and Computer Engineering at the University of British Columbia.\\
+    FPGA & Field-Programmable-Gate-Arrays, a ``programmable'' hardware that allows ASIC-like performance with software-like turn-around time and flexibility.\\
+    GPU & Graphics Processing Unit, a discrete piece of hardware designed to accelerate graphic-intensive or other parallel computing tasks.\\
+    LOS & Line-of-sight.\\
+    ML & Machine learning.\\
+    Multirotor & An unmanned vehicle with multiple engines. \\
+    OTS & Off-the-shelf, or commercially available/purchaseable \\
+    PID / PID Controller & Proportion-Integral-Derivative controllers is the most common control algorithm for precise and accurate movement, as well as to compensate external forces.\cite{pid}\\
```

Comment:
Are these an algorithm or a device?

Reply:
I would say it's more of an algorithm. Not a software specific algorithm, but a general algorithm